source/educators/how-tos/course_development/configure_resources.rst

```diff
@@ -32,17 +31,17 @@ To change how the discussion experience is configured, follow these steps:
 #. Select **Apply** to save your configuration changes.
 
 The discussion configuration experience is comprised of two steps: provider selection and provider configuration.
-By default, courses are created with the edX discussion experience and its default configuration.
+By default, courses are created with the discussion experience and its default configuration.
```

Comment:
```suggestion
By default, courses are created with the Open edX discussion experience and its default configuration.
```

There are multiple different types of discussion you can use, like YellowDig or piazza, so good to specify the name of the default.